.github/workflows/templates/build-wheel-release-upload.yml

```diff
@@ -11,6 +11,8 @@ jobs:
     uses: Billingegroup/release-scripts/.github/workflows/_build-wheel-release-upload.yml@{{ VERSION/v0 }}
     with:
       project: {{ PROJECT/PROJECT_NAME }}
+      c_extension: {{ C_EXTENSION/false }}
+      github_admin_username: 'sbillinge'
```

Comment:
this is a template - since we don't have the user's github username, we hardcode `sbillinge` for now but if we decide to the user's info (https://github.com/Billingegroup/cookiecutter/issues/208#issuecomment-2564047037) then this can be populated dynamically.